enterprise/cli/licenses.go

```diff
@@ -0,0 +1,114 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+
+	agpl "github.com/coder/coder/cli"
+	"github.com/coder/coder/cli/cliui"
+	"github.com/coder/coder/codersdk"
+)
+
+var jwtRegexp = regexp.MustCompile(`^[A-Za-z0-9_-]+\.[A-Za-z0-9_-]+\.[A-Za-z0-9_-]+$`)
+
+func licenses() *cobra.Command {
+	cmd := &cobra.Command{
+		Short:   "Add, remove, and list licenses",
```

Comment:
This description doesn't fit what this presently does, so if we did a release this might confuse people.

Reply:
That's true, but I am working on those, and the converse risk is forgetting to update this